CrossPlatformDesktopProject/CrossPlatformDesktopProject/Link/Equipables/Boomerang.cs

```diff
@@ -12,7 +13,7 @@
 
 namespace CrossPlatformDesktopProject.Equipables
 {
-    class Boomerang : IEquipable
+    class Boomerang : IEquipable, ICollider

```

Comment:
Could make IEquipable implement ICollider so that the equipable classes do not all need to have the `, ICollider` part (matches what was done with other collidable interfaces)

Reply:
But looking at the bomb example, the bomb shouldn't collide, only the smoke.

Reply:
That makes sense, your way is probably better then